src/components/MapView/Legends/index.tsx

```diff
@@ -228,7 +228,11 @@ function LegendItem({
                 legend.map(({ value, color }: any) => (
```

Comment:
know this isn't in scope for this PR, but would love if we could investigate why there's an `any` here. Possibly a comment or fix would be good :+1:

Reply:
Included in 56e98bb6b12cf499da432ececb7b027b616a31d9